src/core/Makefile.am

```diff
@@ -183,6 +184,7 @@ noinst_HEADERS                      = \
     crypto/aes_ecb.hpp                \
     crypto/hmac_sha256.hpp            \
     crypto/mbedtls.hpp                \
+    crypto/pbkdf2_cmac.h              \
```

Comment:
@nibanks, Thanks for the suggestion, I have added the new files to VS project files. But the AppVeyor build still fails on platform x64 due to the warning: conversion from 'size_t' to 'int', the problem codes locate in mbedtls subtree.

Is there any way to workaround this build failure? Thanks.